mantidimaging/gui/windows/live_viewer/view.py

```diff
@@ -52,6 +53,10 @@ def watch_directory(self) -> None:
         """Show the most recent image arrived in the selected directory"""
         self.presenter.set_dataset_path(self.path)
 
+    def load_exisitng_images(self) -> None:
+        """Load the existing images in the selected directory if present"""
+        self.presenter.model.get_images()
+
```

Comment:
This is not used. I'd say its fine to call the `get_images()` directly than to have a pass through function.

Reply:
I also considered adding directly to ImageWatcher, but couldn't find a nice way to call `self._handle_directory_changes()` that didn't cause `TypeError: native Qt signal is not callable` when called from within the same class and add duplication of `_handle_image_changed_in_list` as `self.images` is needed from `LiveViewerWindowModel` to include the latest images.